aleph_alpha_client/completion.py

```diff
@@ -156,9 +176,13 @@ class CompletionRequest(NamedTuple):
     sequence_penalty: float = 0.0
     sequence_penalty_min_length: int = 2
     use_multiplicative_sequence_penalty: bool = False
+    completion_bias_inclusion: Optional[Sequence[str]] = None
+    completion_bias_inclusion_first_token_only: bool = False
+    completion_bias_exclusion: Optional[Sequence[str]] = None
+    completion_bias_exclusion_first_token_only: bool = False
 
     def to_json(self) -> Dict[str, Any]:
-        payload = self._asdict()
```

Comment:
was this change necessary at all? Or is it more like an optimization?

Reply:
On the python side an Option[Sequence] make sense as you typically do not want to have a mutable default value (i.e. []), however `None` gets serialized to a `null` json. On the API side there is no reason to support a _nullable_ field as a non-existing field can simply be deserialized into the default empty list. This change makes sure that we do not send `null`s in json structs for _undefined_ fields. Makes sense?

Reply:
👍